internal/cmd/skupper/site/nonkube/site_delete_test.go

```diff
@@ -27,9 +27,9 @@ func TestCmdSiteDelete_ValidateInput(t *testing.T) {
 		expectedError     string
 	}
 
-	homeDir, err := os.UserHomeDir()
+	err := os.Setenv("SKUPPER_OUTPUT_PATH", "/tmp/skupper")
```

Comment:
I am not super confident this actually addresses the issue - feels to me like it just _moves_ the problem to `/tmp`.
I'd suggest a per-test temp directory so that parallel executions do not clobber each other. the testing package as a method for this: https://pkg.go.dev/testing#T.TempDir

Secondarily, setting environment variables in tests can be dicey as well - lots of low level os thread and global mutable state problems can pop up. Maybe there's a way to factor that out and pass the path in to the command setup rather than pulling from os.Environ? If that's too much of a bear, I think it's okay for now, but if it is still flakey with a random temp dir that's where i'd look.

Reply:
changed!

I think that the refactor of the command would take a little bit of thinking, but if this happens again I would definitely refactor that.